scripts/dev/e2e.py

```diff
@@ -69,18 +71,6 @@ def _prepare_test_environment(config_file: str) -> None:
     )
 
 
-def _delete_test_pod(config_file: str) -> None:
-    """
-    _delete_testrunner_pod deletes the test runner pod
-    if it already exists.
-    """
-    dev_config = load_config(config_file)
-    corev1 = client.CoreV1Api()
-    k8s_conditions.ignore_if_doesnt_exist(
-        lambda: corev1.delete_namespaced_pod(TEST_POD_NAME, dev_config.namespace)
-    )
-
```

Comment:
I moved _delete_test_pod down so that it is below _delete_test_environment because create_test_pod is also below _create_test_environment